CMakeLists.txt

```diff
@@ -83,3 +83,20 @@ target_compile_options(
     piper_phonemize_exe PUBLIC
     ${ESPEAK_NG_CFLAGS_OTHER}
 )
+
+# ---- Declare install targets ----
+
+install(
+    TARGETS piper_phonemize
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+install(
+    DIRECTORY ${CMAKE_SOURCE_DIR}/src/
+    DESTINATION include/piper-phonemize
+    FILES_MATCHING
+    PATTERN "*.h"
+    PATTERN "*.hpp")
+
+install(
+    TARGETS piper_phonemize_exe
```

Comment:
installing an executable to libdir does not look correct, unless i'm not reading this correctly

Reply:
That is indeed correct! During tests CMake installed it to [..]/bin anyways, but I've updated it now to `${CMAKE_INSTALL_DIR}`

Reply:
did you mean `CMAKE_INSTALL_BINDIR`?

Reply:
Yes, corrected. Sorry for the iteration

Reply:
nothing to be sorry for! i don't know much cmake myself